### PR TITLE
feat(package): add repository information to package.json

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**
Signed-off-by: Aniruddha Shriwant <aniruddhashriwant@gmail.com>

# Closes #[341](https://github.com/accordproject/techdocs/issues/341)<CORRESPONDING ISSUE NUMBER>

### Changes

-  Added Package Repository Information (directory) in `package.json` <ONE>

### CODE REVIEW

File: packages/dingus/package.json [hunk 1]
```diff
@@ -8,7 +8,11 @@
     "markdown-it",
     "commonmark"
   ],
-  "repository": "accordproject/markdown-transform",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/accordproject/markdown-transform.git",
+    "directory": "packages/dingus"
+  },
```
Comment (irmerk) [inline packages/dingus/package.json:15]: I don't believe we want to worry about the dingus in this, @jeromesimeon thoughts?

File: packages/markdown-slate/package.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/accordproject/slate-markdown-transform.git"
+    "url": "https://github.com/accordproject/slate-markdown-transform.git",
```
Comment (irmerk) [inline packages/markdown-slate/package.json:27]: I believe this is an outdated link.

Comment (Aniruddha-Shriwant) [reply packages/markdown-slate/package.json]: Sorry for that, I had changed the outdated link

File: packages/dingus/package.json (hunk 1 above)
Comment (Aniruddha-Shriwant) [reply packages/dingus/package.json:15]: So should I revert changes for this file?

**Comment by irmerk** [comment]
Please don't merge until all related PRs are approved (tracked in https://github.com/accordproject/techdocs/issues/341)

File: packages/dingus/package.json (hunk 1 above)
Comment (dselman) [reply packages/dingus/package.json:15]: I think it's fine to leave it - no harm.

**Review by dselman — APPROVED** [review]
Thank you!

**Pull request merged**